addons/web/static/src/views/fields/progress_bar/progress_bar_field.js

```diff
@@ -138,7 +144,7 @@ export const progressBarField = {
     extractProps: ({ attrs, options }) => ({
         maxValueField: options.max_value,
         currentValueField: options.current_value,
-        isEditable: !options.readonly && options.editable,
+        isEditable: options.editable,
```

Comment:
why this ? I grepped in the code, and indeed, this combination is not present, but is this legit ?

Reply:
We can keep it of course. It just found it confusing and unused when I was looking at options